api/v1/imagelist_types.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
v1 and v1alpha1 should have zz_conversion files. make sure to add the conversion-gen markers and `make generate`

Reply:
done